python/lsst/ctrl/stats/data/dbEntry.py

```diff
@@ -33,15 +32,33 @@ def __init__(self, dbList):
         """
         # the DAG node name
```

Comment:
I fixed this, and all the rest of the docstrings that were in __init__ instead of the class docstring.